presto-main/src/test/java/com/facebook/presto/operator/TestHashJoinOperator.java

```diff
@@ -914,7 +1016,7 @@ private BuildSideSetup setupBuildSide(
                 partitionCount,
                 new PagesIndex.TestingFactory(),
                 spillEnabled,
-                SINGLE_STREAM_SPILLER_FACTORY);
+                singleStreamSpillerFactory);
```

Comment:
It looks like you started with separating preparatory changes but then you gave up and up everything in the last commit. For example, you could have this change, or changing LJO processing logic from "call it directly" to "wrap it with a driver" as a yet another preparatory commit.